scout/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '4.1.4'
```

Comment:
Please don't update version in pull requests. I will make a release from master when it is merged

Reply:
Ok!